arangod/Cluster/ActionBase.cpp

```diff
@@ -170,11 +170,8 @@ void ActionBase::startStats() {
 
 /// @brief show progress on Action, and when that progress occurred
 void ActionBase::incStats() {
-  auto progress = _progress.load();
-  progress += 1.;
-  _progress.store(progress);
+  _progress.fetch_add(1.);
```

Comment:
We are using atomic increment here, whereas in 3.10 & 3.11 we are increasing the value under a mutex.
This is ok for me, because the reason to use a mutex in 3.10 & 3.11 was that the atomic increment did not compile on macOS. This should not be a problem for our devel branch, but wanted to confirm here that this intentionally deviates from 3.10 & 3.11.

Reply:
This was exactly my understanding, that we no longer need to support the Mac compiler in 3.12.

Reply:
We support macOS compilation for `client-tools` target only.